prep-javascript-variables/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Marcio Chiaradia';
+    console.log('value of fullName: ', fullName);
+
+    var numberOfPets = '0';
+    console.log('value of numberofPets: ', numberOfPets);
+
+    var likesVegetables = Boolean();
```

Comment:
Hi Marcio, here you can use `true` or `false` boolean literals. W3Schools is kinda whack with their example.